### PR TITLE
Bug 1221121: check input scopes on createRole

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: auth/v1.js [hunk 1]
```diff
@@ -427,15 +427,19 @@ api.declare({
...
+    "If there already exists a role with the same `roleId`this operation",
+    "will fail. Use `updateRole` to modify an existing role.",
   ].join('\n')
 }, async function(req, res) {
   let roleId    = req.params.roleId;
   let input     = req.body;
 
   // Check scopes
-  if (!req.satisfies({roleId})) {
+  if (!req.satisfies({roleId}) !! !req.satisfies([input.scopes]) {
```
Comment (petemoore) [inline auth/v1.js]: should `!!` be `||`?

Comment (djmitche) [inline auth/v1.js]: oh, geez, yes

Comment (petemoore) [inline auth/v1.js]: haha, well no worries, we should probably make getting the CI tests working in travis a priority so these things would get picked up.

**Comment by jonasfj** [comment]
nice catch... thanks...

**Comment by jonasfj** [comment]
hmm... This broke my tests... he he... fixed and landing...

**Pull request merged**